jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/tasklogs/TaskLogsPresenterTest.java

```diff
@@ -229,12 +232,8 @@ public void loadMoreLogs() {
         return summaryList;
     }
 
-    private Date createDate(int year,
-                            Month month,
-                            int day) {
-        LocalDate localDate = LocalDate.of(year,
-                                           month,
-                                           day);
+    private Date createDate(int year, Month month, int day) {
```

Comment:
@jsoltes I see code formatting problems, I think we have to synchronize urgently this because this introduce a style, different from the one we are applying.
Here is what I'm using, and I see Cristiano is following the same
https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml 
Maybe @cristianonicolai could give us some light here

Reply:
@nmirasch regarding the code style. I am using the codestyles from droolsjbpm-build-bootstrap (from the latest master). I allows both inline and line-separate parameters. I actually changed this manually as I prefer this, but I am going to change it back as you probably prefer the other style (and there is no reason for me to change it other that it annoys me a little). I prefer to have less lines of code when the parameters are short. So my point is, the codestyle does not require one or the other signature, it is just a matter of personal preference...

Reply:
@jsoltes I think it's not a matter of personal preference (more if you have to change it manually). IMO makes sense follow the same style along the code, To ensure that, sometime ago we agreed to configure the IDE to do 'Reformat code' before commit. But we can talk this on Monday at sync meeting, no worries :)

Reply:
@jsoltes @nmirasch I personally think that the line wrapping for variables isn't necessary but is probably best if we follow the same guidelines as the rest of the team, so sticking to what is defined in the formatter.